app/utils/mappers/AmendIndividualBeneficiaryMapper.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.mappers
+
+import java.time.LocalDate
+
+import models.beneficiaries.IndividualBeneficiary
+import models.{Address, CombinedPassportOrIdCard, IndividualIdentification, Name, NationalInsuranceNumber, NonUkAddress, UkAddress, UserAnswers}
+import org.slf4j.LoggerFactory
+import pages.individualbeneficiary._
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsError, JsSuccess, Reads}
+
+class AmendIndividualBeneficiaryMapper {
+
+  private val logger = LoggerFactory.getLogger("application." + this.getClass.getCanonicalName)
+
+  def apply(answers: UserAnswers): Option[IndividualBeneficiary] = {
+    val readFromUserAnswers: Reads[IndividualBeneficiary] =
```

Comment:
Could this be simplified to:

`val readFromUserAnswers: Reads[IndividualBeneficiary] =
      (
        NamePage.path.read[Name] and
          DateOfBirthPage.path.readNullable[LocalDate] and
          readIdentification and
          readAddress and
          VPE1FormYesNoPage.path.read[Boolean] and
          readIncome and
          IncomeDiscretionYesNoPage.path.read[Boolean] and
          StartDatePage.path.read[LocalDate] and
          Reads(_ => JsSuccess(true))
        ) .apply(IndividualBeneficiary.apply _ )`

Reply:
No as we need to default the provisional flag when we instantiate an instance of IndividualBeneficiary()

Reply:
The above code defaults it to true no?